packages/react-native-bpk-component-text-input/src/theming.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// @flow
+
+import { makeThemePropType } from 'react-native-bpk-theming';
+
+const REQUIRED_THEME_ATTRIBUTES = ['textFontFamily', 'textInputFocusedColor'];
```

Comment:
Perhaps we should export `REQUIRED_THEME_ATTRIBUTES` from `BpkText` and then import it here so it wouldn't need to be duplicated, or do you think that's overkill?

Reply:
I don't think we should at this point, we might want to have different props for text and input. Also even if we import it here we would still need to make use of them in the code so it wouldn't give us any benefit right now.

Reply:
Ok fair enough 👍